enclone_print/src/print_utils4.rs

```diff
@@ -495,11 +497,10 @@ pub fn compute_bu(
     exact_clonotypes: &[ExactClonotype],
     row: &mut [String],
     subrows: &mut Vec<Vec<String>>,
-    varmat: &[Vec<Vec<u8>>],
     have_gex: bool,
     gex_info: &GexInfo,
     rsi: &ColInfo,
-    sr: &mut Vec<(Vec<String>, Vec<Vec<String>>, Vec<Vec<u8>>, usize)>,
```

Comment:
Good that we got rid of two unused fields here

Reply:
And gigantic ones at that!